### PR TITLE
lib/utils: Port a bit to decl-after-stmt style 

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
OK, this patch grew in scope.

**Comment by jlebon** [comment]
Let's split out the style porting into a seperate PR?

**Comment by cgwalters** [comment]
There's conflicts though.  I guess a more proper order would be the `mmap()` first, then style?

**Comment by cgwalters** [comment]
OK, this is back to just style porting, but let's do `mmap()` first and I'll rebase this one on it?

File: src/libotutil/ot-variant-utils.c [hunk 1]
```diff
@@ -36,45 +37,42 @@ ot_gvariant_new_empty_string_dict (void)
...
 {
-  gpointer data_copy;
-  GVariant *ret;
-
-  data_copy = g_memdup (data, len);
-  ret = g_variant_new_from_data (G_VARIANT_TYPE ("ay"), data_copy,
+  gpointer data_copy = g_memdup (data, len);
+  GVariant *ret = g_variant_new_from_data (G_VARIANT_TYPE ("ay"), data_copy,
                                  len, FALSE, g_free, data_copy);
   return ret;
```
Comment (jlebon) [inline src/libotutil/ot-variant-utils.c]: Minor: we can just `return` it directly right, right?

**Comment by jlebon** [comment]
@rh-atomic-bot r+ f8adf9d

**Pull request closed without merging**